components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -736,18 +809,37 @@ public RuleTemplate getRuleTemplate(BusinessRuleFromTemplate businessRuleFromTem
      * Saves JSON definition of the given Business Rule, to the database
      *
      * @param businessRuleFromTemplate
-     * @throws TemplateManagerException
+     * @throws TemplateManagerException,UnsupportedEncodingException,BusinessRulesDatasourceException,SQLException
      */
     public void saveBusinessRuleDefinition(String uuid, BusinessRuleFromTemplate businessRuleFromTemplate, boolean
             isDeployed) throws
-            TemplateManagerException {
+            TemplateManagerException, UnsupportedEncodingException, BusinessRulesDatasourceException, SQLException {
+        QueryExecutor queryExecutor = new QueryExecutor();
+        int deploymentState = 0;
+        byte[] businessRule=businessRuleFromTemplate.toString().getBytes("UTF-8");
```

Comment:
Format the code properly !!

Reply:
Fixed by 47e9832228beaa5747e074303ac9d729cc69abdf